apps/web/.env.local

```
@@ -0,0 +1,15 @@
+# Environment Variables Example
+# Copy this file to .env.local and update the values as needed
+
+DATABASE_URL="postgresql://opencut:opencutthegoat@localhost:5432/opencut"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)